File: 03/12_test.py
class Test:
    other = None
    msg = "hello\nworld"
    
    def getOther(self):
        return self.other
    
    def compute(self, obj: Test):
        self.other = obj.getOther()
        return self.compute(obj)
    
    def empty(self):
        pass

File: 03/12_test.java
public class Test {
    private Test other = null;
    private String msg = "hello\nworld";

    public Test getOther() {
        return other;
    }

    public int compute(Test obj) {
        other = obj.getOther();
        return compute(obj);
    }

    public void empty() {
    }
}